src/core/Microsoft.PowerToys.Settings.UI.Lib/SettingsUtils.cs

```diff
@@ -67,7 +67,7 @@ public static void SaveSettings(string jsonSettings, string powertoy)
             }
         }
 
-        private static string LocalApplicationDataFolder()
+        public static string LocalApplicationDataFolder()
```

Comment:
Should we use some other folder to create files in case of unit tests instead of Local App data folder ? The reason being we end up messing up some files in that folder or may leave some orphan files after Unit tests execution.

Reply:
Create the files in a different folder would not solve the problem. I added code to remove the folder containing test files after each test: 
https://github.com/microsoft/PowerToys/pull/2129/commits/1756d332fbe00938d366b0ea37147604660814bb